var/spack/repos/builtin/packages/neko/package.py

```diff
@@ -15,6 +15,7 @@ class Neko(AutotoolsPackage, CudaPackage, ROCmPackage):
     git      = "https://github.com/ExtremeFLOW/neko.git"
     maintainers = ['njansson']
 
+    version('0.3.2', commit='e7e43fb0f766e8a92e053ec48e140c7449098f41')
     version('0.3.0', commit='a82097a10ae3c965fb873da909a6324c7a7742fb')
```

Comment:
I confirmed the version commit but wanted to note that Spack generally prefers use of hashes when possible.  

IF you're interested in switching, the following changes are an option:

```suggestion
    url = https://github.com/ExtremeFLOW/neko/releases/download/v0.3.2/neko-0.3.2.tar.gz
    maintainers = ['njansson']

    version('0.3.2', sha256='0628910aa9838a414f2f27d09ea9474d1b3d7dcb5a7715556049a2fdf81a71ae')
    version('0.3.0', sha256='e46bef72f694e59945514ab8b1ad7d74f87ec9dca2ba2b230e2148662baefdc8')
```

where the `sha256` values are obtained from running `spack checksum neko 0.3.2 0.3.0` in this case (and only after adding the URL (pattern)).

Reply:
Thanks for the suggestion. We will switch to `sha256`.